benefit/data/sheets/AuthorisationCaseField/AuthorisationCaseField.json

```diff
@@ -366,6 +367,7 @@
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "tcwInProgress", "UserRole": "caseworker-sscs-superuser", "CRUD": "CRUD"},
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "evidenceReceivedCF", "UserRole": "caseworker-sscs-superuser", "CRUD": "CRUD"},
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "caseBundles", "UserRole": "caseworker-sscs-superuser", "CRUD": "CRUD"},
+  {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "historicalBundles", "UserRole": "caseworker-sscs-superuser", "CRUD": "CRUD"},
```

Comment:
This line is not needed since the role derives from caseworker-sscs which has CRUD access